todo_list/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for todo_list project.
+Generated by 'django-admin startproject' using Django 4.0.4.
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-r#4o00zb3g*%diq!@(qe7n*u4(u6z!$z@3cm7ch*npwr=@qr7s"
```

Comment:
Secret keys and other secure info you should store at environment variables.
You can read more info about it here [https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1](url) or just google it by yourself, since this topic is really important